site/_includes/explanations/permissions.md

```diff
@@ -14,6 +14,37 @@ $ pulsar-admin namespaces grant-permission test-property/cl1/ns1 \
   --role admin10
 ```
 
+Wildcard authorization can be performed when `authorizationAllowWildcardsMatching` is set to `true` in `broker.conf`.
+
+e.g.
+```shell
+$ pulsar-admin namespaces grant-permission test-property/cl1/ns1 \
+                        --actions produce,consume \
+                        --role my.role.*
```

Comment:
Should this be written with quotes to avoid Bash from doing file pattern matching?

```shell
...
--role 'my.role.*'
```